serving/docs/lmi_new/large_model_inference.md

```diff
@@ -0,0 +1,67 @@
+# Table of Contents
+- [Overview - Large Model Inference (LMI) Containers](#overview---large-model-inference-lmi-containers)
+- [QuickStart](#quickstart)
+  - [Sample Notebooks](#sample-notebooks)
+  - [Deployment Guide](#deployment-guide)
+- [Components of LMI](#components-of-lmi)
+- [Support LMI Inference Backends](#supported-lmi-backends)
+
+# Overview - Large Model Inference (LMI) Containers
+
+LMI containers are a set of high performance Docker Containers purpose built for large language model (LLM) inference. 
+These containers allow users to easily deploy LLMs as self-hosted endpoints using Amazon SageMaker. 
+LMI containers enable users to leverage the latest Open Source technologies to deploy popular Open Source foundation models. 
+These containers bundle together a model server with open source inference stacks to deliver an all-in-one LLM serving solution. 
+LMI containers provide many features, including:
+
+* Optimized inference performance for popular model architectures like Llama, Bloom, Falcon, T5, GPTNeo and more
+* Integration with open source inference frameworks like VLLM, TensorRT LLM, DeepSpeed, and Transformers NeuronX
+* Iterative Batching for maximizing throughput at high concurrency
+* Token Streaming
+* Quantization through AWQ, GPTQ, and SmoothQuant
+* Multi GPU inference using Tensor Parallelism
+* Serving LoRA fine-tuned models
+
+## QuickStart
+
+### Sample Notebooks
+The following table provides example notebooks for how to deploy popular open source LLMs using LMI containers on SageMaker.
+For each model, our recommended stack is based on benchmarking the model across supported stacks using the same instance type, prompt, concurrency, and generation parameters.
+If this is your first time using LMI, or you want a starting point for deploying a specific model, we recommend following the notebooks below.
+
+TODO: Add Table
+
+### Deployment Guide
+
+We have put together a comprehensive [deployment guide](deployment_guide.md) that takes you through the steps needed to deploy a model using LMI containers on SageMaker.
+The document covers the phases from storing your model artifacts through benchmarking your SageMaker endpoint.
+
+## Components of LMI
```

Comment:
Should this come before the quick-start? I think we wanted to have the sample quick-start notebooks near the top to get users deploying quicker rather than bogged down in the technical details.